config.md

```diff
@@ -28,10 +28,12 @@ For example, if a configuration is compliant with version 1.1 of this specificat
 
 **`root`** (object, REQUIRED) specifies the container's root filesystem.
 
-* **`path`** (string, REQUIRED) Specifies the path to the root filesystem for the container.
+* **`path`** (string, OPTIONAL) Specifies the path to the root filesystem for the container.
   The path is either an absolute path or a relative path to the bundle.
-  On Linux, for example, with a bundle at `/to/bundle` and a root filesystem at `/to/bundle/rootfs`, the `path` value can be either `/to/bundle/rootfs` or `rootfs`.
-  A directory MUST exist at the path declared by the field.
+  On Linux, for example, with a bundle at `/to/bundle` and a root filesystem at `/to/bundle/rootfs`, the `path` value can be either `/to/bundle/rootfs` or `rootfs`. 
```

Comment:
Travis doesn't like [the trailing whitespace here][1]:

```
$ git show --oneline --check origin/pr/820
23a0b15 Clarify root path
config.md:33: trailing whitespace.
+  On Linux, for example, with a bundle at `/to/bundle` and a root filesystem at `/to/bundle/rootfs`, the `path` value can be either `/to/bundle/rootfs` or `rootfs`. 
```

[1]: https://travis-ci.org/opencontainers/runtime-spec/jobs/231771631#L355